aapcs64/aapcs64.rst

```diff
@@ -1787,13 +1790,13 @@ set to zero, permitting a simplified implementation of ``va_arg`` which simply a
 
 Although this standard does not mandate a particular stack frame organisation beyond what is required to meet the stack constraints described in `The Stack`_, the following figure illustrates one possible stack layout for a variadic routine which invokes the ``va_start`` macro.
 
-.. figure:: aapcs64-variadic-stack.png
+.. figure:: aapcs64-variadic-stack.svg
```

Comment:
No, that wasn't on purpose, and I'm quite confused, because I _thought_ I was being careful about those prime signs!

I agree that they're right the original way round; in particular, the FP' stored on the stack in the new stack frame on the RHS ought to be the same as the one before the call on the LHS.